examples/catalog/lib/animated_list.dart

```diff
@@ -21,14 +21,19 @@ class _AnimatedListSampleState extends State<AnimatedListSample> {
     super.initState();
     _list = ListModel<int>(
       listKey: _listKey,
-      initialItems: <int>[0, 1, 2],
+      initialItems: <int>[
+        0,
+        1,
+        2
+      ],
```

Comment:
I think this one should be left alone. I don't think it makes sense to split something if it all fits on one line.

Reply:
is there a dartfmt bug filed to fix this case?

Reply:
Resolving this. This was a now fixed bug I introduced as a noob formatter hacker.